sass/form/input-textarea.sass

```diff
@@ -45,6 +47,10 @@
 
 .textarea
   @extend %input-textarea
+  -webkit-appearance: none
+  box-shadow: none
+  border-width: 1px
+  border-style: solid
```

Comment:
You're duplicating a lot of properties. These are already applied by `%input-textarea`.